site/en/docs/extensions/mv3/tut_oauth/index.md

```diff
@@ -11,12 +11,11 @@ description: >
 
 {% include 'partials/extensions/mv2page-in-mv3.md' %}
 
-[OAuth2][1] is the industry-standard protocol for authorization. It provides a mechanism for users
+[OAuth2][oauth2] is the industry-standard protocol for authorization. It provides a mechanism for users
 to grant web and desktop applications access to private information without sharing their username,
 password and other private credentials.
 
-This tutorial builds an extension that accesses a user's Google contacts using the [Google People
-API][2] and the [Chrome Identity API][3]. Because extensions don't load over HTTPS, can't perform
+This tutorial builds an extension that accesses a user's Google contacts using the [Google People API][people-api] and the [Chrome Identity API][identity-api]. Because extensions don't load over HTTPS, can't perform
 redirects or set cookies, they rely on the Chrome Identity API to use OAuth2.
```

Comment:
```suggestion
This tutorial builds an extension that accesses a user's Google contacts using the [Google People
API][people-api] and the [Chrome Identity API][identity-api]. Because extensions don't load over
HTTPS, can't perform redirects or set cookies, they rely on the Chrome Identity API to use OAuth2.
```

Wrap at 100 characters